sys/include/ztimer/config.h

```diff
@@ -114,6 +121,13 @@ extern "C" {
 #  endif
 #endif
 
+/**
+ * @brief   The minimum pm mode required for ZTIMER_LPTIMER to run
+ */
+#ifndef CONFIG_ZTIMER_LPTIMER_BLOCK_PM_MODE
+#  define CONFIG_ZTIMER_LPTIMER_BLOCK_PM_MODE ZTIMER_CLOCK_NO_REQUIRED_PM_MODE
```

Comment:
do we do the spacing with defines?

Reply:
I just tried to adapt the surrounding indentation.

The naming has been picked by the already merged PR introducing `ztimer_periph_lptimer`.